From 34172ae4eec2208476b31edf79d93e7b516ccd24 Mon Sep 17 00:00:00 2001
From: Riddhi Aggarwal <quic_riddagga@quicinc.com>
Date: Mon, 24 Apr 2023 13:48:14 +0530
Subject: [PATCH 207/281] arm64: dts: qcom: ipq9574: Adding build support for
 various RDPs

Adding build support for the following RDPs
rdp 418
rdp 454
rdp 455
rdp 456
rdp 467
rdp 469

Change-Id: Idbaa350ed2eb7d9aa0dcbdef477be0c90ed84b27
Signed-off-by: Riddhi Aggarwal <quic_riddagga@quicinc.com>
---
 arch/arm64/boot/dts/qcom/Makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 40c66e5fb834..4e71a679c969 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -9,7 +9,13 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq6018-cp01-c1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk01.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk10-c1.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq8074-hk10-c2.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp418.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp433.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp454.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp455.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp456.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp467.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp469.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-alcatel-idol347.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-asus-z00l.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-huawei-g7.dtb
-- 
2.17.1

